Source/NSCollectionView.m

```diff
@@ -622,6 +622,12 @@ - (id) initWithCoder: (NSCoder *)aCoder
         }
       else
         {
+          _minItemSize = [aCoder decodeSize];
+          _maxItemSize = [aCoder decodeSize];
+          [aCoder decodeValueOfObjCType: @encode(int64_t) at: &_maxNumberOfRows];
```

Comment:
As you may remember we just decided on not using explicit sized integer variants in coding. Could you please instead use NSUInteger and maybe even better use the new macros from GSGuiPrivate.h?

Reply:
Okay, I will change that.  Thx